### PR TITLE
Fix static resources on Plone 5.2+

### PR DESCRIPTION
**Pull request by laulaz**
This related to issue #20, and ensure compatibility with Plone 4 & 5+ :
 - use Resource Registry to register CSS & JS for Plone 5+
 - keep using cssregistry.xml and jsregistry.xml for Plone 4

Build(out) has been fixed for Python 3, but not for Python 2. It was already broken, this is not related to this change (https://travis-ci.org/github/collective/collective.faq/builds/771556189).

@tisto could you review this please ?

### CODE REVIEW

File: src/collective/faq/configure.zcml [hunk 1]
```diff
@@ -15,14 +15,32 @@
...
+      name="install-base"
       title="FAQ (collective.faq)"
-      directory="profiles/default"
+      directory="profiles/base"
       description="Installs the FAQ add-on."
       provides="Products.GenericSetup.interfaces.EXTENSION"
       post_handler=".setuphandlers.post_install"
       />
 
+  <genericsetup:registerProfile
```
Comment (tisto) [inline src/collective/faq/configure.zcml]: @laulaz wondering if we need an upgrade step here when we switch the gs profile architecture.

Comment (laulaz) [reply src/collective/faq/configure.zcml]: @tisto I asked myself exactly the same question and though that if `collective.faq` was already installed, as I didn't change the profile version, it wouldn't hurt (and "default" profile still exists).

But I haven't tested all the upgrade paths.

I could write an upgrade step for Plone 5.0 & 5.1 (versions where we come from [css/js]registry to Resource Registry) (?) ... or maybe we go on and drop support for Plone 4 on this branch :innocent: :wink:

Comment (tisto) [reply src/collective/faq/configure.zcml]: @laulaz I guess we should rather drop Plone 4 support on this branch. It is not worth the effort and we will have to drop it sooner or later. I am happy to cut a new Plone 5.2 / Python 3 only release.

Comment (laulaz) [reply src/collective/faq/configure.zcml]: @tisto OK, I will propose the according changes 👍

**Comment by laulaz** [comment]
@tisto looks good to you ?

I made the following choices :
 - version bump from 1.2.x (< Plone 5.2) to 1.3.x (>= Plone 5.2). I will of course create the related 1.2.x branch
 - fresh profile version & remove of existing upgrades, as we will not handle migration to this new release
 - no change to buildout files (nor removal of previous Plone versions) because they can be updated by the Makefile

File: CHANGES.rst [hunk 2]
```diff
@@ -2,10 +2,14 @@ Changelog
 =========
 
 
-1.2.1 (unreleased)
+1.3.0 (unreleased)
```
Comment (tisto) [inline CHANGES.rst]: @laulaz I'd go with a 2.0.0 since this is a breaking change.

File: .travis.yml [hunk 3]
```diff
@@ -1,7 +1,6 @@
 language: python
 os: linux
 python:
-  - "2.7"
```
Comment (tisto) [inline .travis.yml:4]: @laulaz we might want to move to use gh actions before we merge this PR. I think Travis for OS projects is dead anyways...

**Comment by laulaz** [comment]
@tisto version 2.0.0 and Github actions ✅

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+      # python cache
+      - uses: actions/cache@v2
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
+
+      # python install
+      - run: pip install virtualenv
```
Comment (tisto) [inline .github/workflows/ci.yml:33]: @laulaz I think we don't need virtualenv here.

**Review by tisto — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 4 above)
Comment (laulaz) [reply .github/workflows/ci.yml:33]: @tisto indeed! same for `wheel`

**Comment by laulaz** [comment]
@tisto thanks for the review & merge !

**Comment by tisto** [comment]
@laulaz thank you for your work on this! Here is the release: https://pypi.org/project/collective.faq/

**Pull request merged**